pkg/build/strategies/layered/layered.go

```diff
@@ -139,8 +167,14 @@ func (b *Layered) Build(config *api.Config) (*api.Result, error) {
 	b.config.LayeredBuild = true
 	// new image name
 	b.config.BuilderImage = newBuilderImage
-	// the scripts are inside the image
-	b.config.ScriptsURL = "image://" + filepath.Join(getDestination(config), "scripts")
+	// see CreateDockerfile, conditional copy, location of scripts
+	scriptsIncluded := scriptsProvided(filepath.Join(config.WorkingDir, api.UploadScripts))
+	glog.V(2).Infof("Scripts dir has contents %v", scriptsIncluded)
+	if scriptsIncluded {
+		b.config.ScriptsURL = "image://" + filepath.Join(getDestination(config), "scripts")
+	} else {
+		b.config.ScriptsURL = os.Getenv("STI_SCRIPTS_PATH")
```

Comment:
OK, @bparees and I chatted off line.  He wants to switch gears on this one, and I'm amenable.  To simplify things, we'll move to getting rid of the "scripts are in the image" special case logic and always copy the scripts to the scripts working dir.  Then, we are back to always copying the working scripts dir in the layered docker file.

A couple of notes:
a) this parallels a PR @mfojtik has for being able to pull source-code out of an image ... so some synergy there
b) we'll want feedback from @soltysh on all this, since he originally did the "image reader" stuff

Priority wise, this is deemed lower priority for this sprint, but I'll spend a few cycles on this next week while it is all still fresh in my mind.